share/module_check_a_mundo.F

```diff
@@ -1838,6 +1838,26 @@ SUBROUTINE check_nml_consistency
       IF ( oops .GT. 0 ) THEN
          count_fatal_error = count_fatal_error + 1
       END IF
+
+
+!-----------------------------------------------------------------------
+! Did the user ask for too many total MPI tasks.
+!-----------------------------------------------------------------------
+
+      oops = 0
+      IF ( model_config_rec % nproc_x * model_config_rec % nproc_y .GT. 10000 ) THEN
```

Comment:
If we modify the RSL code to dynamically size the names of RSL files, and with a few other modifications, I don't think this sort of check should be needed.